src/pages/LandingPage/index.tsx

```diff
@@ -1,45 +1,21 @@
 // Constants - utils
-import { auth, db, GoogleProvider } from "@/services/firebase";
-import { signInWithPopup } from "firebase/auth";
-import { doc, getDoc, setDoc } from "firebase/firestore";
-import { useAuthState } from "react-firebase-hooks/auth";
+import useAuth from "@/hooks/useAuth";
+import { useNavigate } from "react-router-dom";
 
 export default function LandingPage() {
-	const [user] = useAuthState(auth);
-
-	const signInWithGoogle = async () => {
-		try {
-			const res = await signInWithPopup(auth, GoogleProvider);
-			// before making a new doc, check if doc already exists to prevent updating the old doc
-			const docRef = doc(db, "users", res.user.uid);
-			const docSnap = await getDoc(docRef);
-			if (!docSnap.exists()) {
-				await setDoc(doc(db, "users", res.user.uid), {
-					uid: res.user.uid,
-					email: res.user.email,
-					displayName: res.user.displayName,
-					profilePic: res.user.photoURL,
-				});
-			}
-		} catch (error) {
-			console.error("Error signing in with Google: ", error);
-		}
-	};
-
-	const handleLogout = async () => {
-		try {
-			await auth.signOut();
-		} catch (error) {
-			console.error("Error signing out: ", error);
-		}
+	const { user, login, logout } = useAuth();
+	// Handle navigation
+	const navigate = useNavigate();
+	const handleNavigateToDashboardPage = () => {
+		navigate("/dashboard");
 	};
 
 	return (
 		<>
-			<div>
+			<div className="w-screen min-h-screen flex items-center justify-center">
```

Comment:
There is a weird white board
![image](https://github.com/matthewdinata/dip-ui/assets/97287370/e57c46c7-6b14-48a9-98ce-a0dbd2f38326)